pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -295,6 +301,67 @@ func TestGetBlockHeader_Negative_Integration(t *testing.T) {
 	}
 }
 
+func TestGetTransactionMerkleProof_Integration(t *testing.T) {
+	transactionHash := testData.TxMerkleProof.TxHash
+	blockHeight := testData.TxMerkleProof.BlockHeigh
+
+	expectedResult := &testData.TxMerkleProof.MerkleProof
+
+	for testName, config := range testConfigs {
+		t.Run(testName, func(t *testing.T) {
+			electrum := newTestConnection(t, config.clientConfig)
```

Comment:
Can we get this one out of the loop?

Reply:
Do you mean getting `electrum := newTestConnection(t, config.clientConfig)` out of the 
```
for testName, config := range testConfigs {
...
}
```?

If so, then it's not possible as the electrum client connection must be recreated with every iteration of the loop
so that different server can be tested every time.

Reply:
You are right, nevermind 🤦🏻